bindings/java/src/main/com/apple/foundationdb/KeySelector.java

```diff
@@ -167,7 +167,7 @@ public String toString() {
 	/**
 	 * Returns the {@code or-equal} parameter of this {@code KeySelector}. For internal use.
 	 */
-	boolean orEqual() {
+	public boolean orEqual() {
```

Comment:
Can you update the comment above as well? The internal use sentence should be removed, and probably we should make it match other public documentation by using the `@return` tag.